flask_awscognito/services/cognito_service.py

```diff
@@ -27,7 +29,7 @@ def __init__(
 
     def get_sign_in_url(self):
         quoted_redirect_url = quote(self.redirect_url)
-        state = get_state(self.user_pool_id, self.user_pool_client_id)
+        state = create_state(self.user_pool_id, self.user_pool_client_id, quote(str(self.client_state)))
```

Comment:
I'd let `create_state` function to `quote(str(...))` on its argument

Reply:
I had it there at first, but then needed to have the utility class import the `quote` function from `urllib.parse`. This way is a bit less readable, but seemed overall tidier. I'll make your suggested edit.